spinoso-string/src/lib.rs

```diff
@@ -671,7 +528,7 @@ impl String {
     /// [`from_raw_parts`]: String::from_raw_parts
     #[must_use]
     pub fn into_raw_parts(self) -> RawParts<u8> {
-        RawParts::from_vec(self.buf)
+        RawParts::from_vec(self.inner.into_vec())
```

Comment:
Once again, `into_vec` will consume it, so I assume this is the "most correct" way to do this.

Reply:
yup that's right.